home.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Block-Chainference</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="style_d.css">
+  <script src="main.js" defer></script>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+
+<!--    Page for block chain coference 2023  -->
+
+<body>
+  <!--    bar menu and logo  -->
+  <header >
+    <nav class="nav-bar-grey">
+      <a href="" class="nav-bar-grey-fabook"></a>
+      <a href="" class="nav-bar-grey-twitter"></a>
+      <a href="" class="nav-bar-grey-language">Language</a>
+      <a href="" class="nav-bar-grey-mypage">My Page</a>
+      <a href="" class="nav-bar-grey-logout">Logout</a>
+    </nav>
+    <nav class="nav-bar">
+      <section class="nav-menuContainer">
+        
+        <button class="bugerBtn">
+          <div class="burger-icon line1"></div>
+          <div class="burger-icon line2"></div>
+          <div class="burger-icon line3"></div>
+        </button>
+        <!--burger menu-->
+        <ul class="burger-menu">
+          <div class="burger-menu-style">
+              <li class="burger-menu-exit-button"><button class="exitButton"></button></li>
+              <li class="burger-menu-homePage-button"><a href="home.html" class="burger-menu-home-page">Home Page</a></li>
+              <li class="burger-menu-aboutPage-button"><a href="about.html" class="burger-menu-aboutPage-button">About</a></li>           
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">Join</a></li>
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">Sponsors</a></li>
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">News</a></li> 
+          </div>
+        </ul>
+        <!---->
+        <ul class="desktop-menu">
+          <li><a href="home.html" class="desktop-menu-home">Home</a></li>
+          <li><a href="about.html" class="desktop-menu-about">About</a></li>
+          <li><a href="" class="desktop-menu-about">Join</a></li>
+          <li><a href="" class="desktop-menu-about">Sponsors</a></li>
+          <li><a href="" class="desktop-menu-about">News</a></li> 
+        </ul>        
+      </section>
+      <div class="nav-logo-container">
+        <a href="home.html" class="nav-logo-link">
+          <div class="nav-logo-img"></div>
+            <h2 class="nav-logo">Libra</h2>
+        </a>
+      </div>
+    </nav>
+  </header>
```

Comment:
kindly consider checking on the image below to stick to the given design as much as possible
<img width="751" alt="Screen Shot 2023-05-25 at 07 31 31" src="https://github.com/Zven94/conference-page/assets/60146030/bc54cf4f-29c1-4de0-943c-641687d444e2">